### ISSUE TITLE
ScrollViewer mouse wheel not working

### ISSUE THREAD

**Issue by dbruning** [benruehl/adonis-ui#50]
If you had a docs page called something like "Tips & tricks" or "Gotcha's", you could add this issue that I came across. I'm not sure why it used to work (before using Adonis styles) - probably because something used to have a background but now doesn't.

Symptom: Scrollviewer doesn't scroll with mousewheel after applying Adonis styles
Possible cause: If you have a control inside a scrollviewer, and the control has a PreviewMouseDown event handler, you may need to add some code to the scrollviewer. See [this Stack Overflow answer](https://stackoverflow.com/a/16235785/84898) for details.

(or - feel free to close this issue, at least someone searching for it might find the answer here.)

**Comment by benruehl** [comment]
Thanks for giving knowledge to others. Currently, I am not quite sure what this is about though.

So you seem to have a ScrollViewer with a ListView inside, I guess. At least thats the setup in the StackOverflow thread you mentioned.

```xml
<ScrollViewer>
    <ListView/>
</ScrollViewer>
```

And the issue is that the ScrollViewer does not move its scroll bars when scrolling with the mouse wheel.

But if I create a new WPF project without any dependencies and put some items into the ListView it has this issue already. It scrolls only when dragging the thumb or scrolling directly over the scroll bar. Doesn't seem to have anything to do with AdonisUI. Did I get something wrong here?

Setting a background brush does not seem to change anything as well. When using AdonisUI the outer ScrollViewer even has `Background="Transparent"` which is specified in the [default ScrollViewer style](https://github.com/benruehl/adonis-ui/blob/master/AdonisUI.ClassicTheme/DefaultStyles/ScrollViewer.xaml).

As far as I know, it is the default behavior of ScrollViewers to handle scroll events even if there is nothing to scroll. This is also pointed out in [this blog post](https://serialseb.com/blog/2007/09/03/wpf-tips-6-preventing-scrollviewer-from/). And some controls like ListViews, ListBoxes, TreeViews, etc. make use of internal ScrollViewers that behave this way as well.

Again, please correct me in case I get this all wrong. But I don't see how this is related to AdonisUI. Do you have some XAML code that behaves differently when AdonisUI is included?

**Comment by dbruning** [comment]
My app didn't actually have a listview inside a scrollviewer. It had something far more complex, involving:
- CaliburnMicro bindings
- ItemsControl with DataTemplate
- Items in the ItemsControl were themselves grids, with labels added dynamically via code
- Labels had mouseover triggers to show a highlight background when moused over

I just tried to distill it into a reproducable github repo, but failed to recreate the problem. I'm 100% sure I had to add that PreviewMouseWheel event handler & code in my app after adding Adonis, but I can't create a simple repro to share. Sorry! Maybe close this issue & if someone else hits my super edge case, they'll find this fix and maybe have more insight into the repro.
